libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/NavigationInstructionPlayer.java

```diff
@@ -56,7 +56,7 @@ public void onDestroy() {
   @Override
   public void onStart() {
     requestAudioFocus();
-    instructionQueue.remove();
```

Comment:
You can just do `instructionQueue.poll` which is the empty-safe version of `remove`